get_repo.sh

```diff
@@ -12,28 +12,28 @@ if [[ -z "${RELEASE_VERSION}" ]]; then
     if [[ "${VSCODE_LATEST}" == "yes" ]] || [[ ! -f "insider.json" ]]; then
       UPDATE_INFO=$( curl --silent --fail https://update.code.visualstudio.com/api/update/darwin/insider/0000000000000000000000000000000000000000 )
     else
-      export MS_COMMIT=$(jq -r '.commit' insider.json)
-      export MS_TAG=$(jq -r '.tag' insider.json)
+      MS_COMMIT=$(jq -r '.commit' insider.json); export MS_COMMIT
```

Comment:
You could move the `export` at the end of the file, I think.

Reply:
Yes I believe `MS_COMMIT` `MS_TAG` and `RELEASE_VERSION` could all be exported only once at the bottom. I wasn't sure if you had it exported in one line for readability so I left it that way. I'll move them to the bottom.